Source/Core/VideoBackends/Software/SetupUnit.cpp

```diff
@@ -25,7 +25,6 @@ void SetupUnit::Init(u8 primitiveType)
 
 OutputVertexData* SetupUnit::GetVertex()
 {
-  memset(m_VertWritePointer, 0, sizeof(*m_VertWritePointer));
```

Comment:
Doesn't this change the behavior if you remove it?
What was the warning you're trying to fix here?

Reply:
first, m_vertWritePointer is not a simple structure so that we cannot do memset.

the usage of this function is only one place:
    // transform this vertex so that it can be used for rasterization (outVertex)
    OutputVertexData* outVertex = m_setup_unit.GetVertex();
    TransformUnit::TransformPosition(&m_vertex, outVertex);
    outVertex->normal = {};
    if (VertexLoaderManager::g_current_components & VB_HAS_NRM0)
    {
      TransformUnit::TransformNormal(
          &m_vertex, (VertexLoaderManager::g_current_components & VB_HAS_NRM2) != 0, outVertex);
    }
    TransformUnit::TransformColor(&m_vertex, outVertex);
    TransformUnit::TransformTexCoord(&m_vertex, outVertex, m_tex_gen_special_case);

So all fields is override in the following code path. So I believe we should remove the memset completely. I test the software video rendering, it works fine.